iset.mm

```diff
@@ -55483,8 +55483,8 @@ associative structure (such as a group).  (Contributed by NM,
 
   ${
     $d A x y $.  $d B x y $.  $d F x y $.
-    $( The ` 2nd ` (second member of an ordered pair) function restricted to a
-       function ` F ` is a function of ` F ` onto the range of ` F ` .
+    $( The ` 2nd ` (second component of an ordered pair) function restricted to
```

Comment:
Although using "member" for ordered pairs is a common usage, I agree that "component" is better. So this change looks good to me.

Reply:
Yes, especially since in set.mm they are encoded as `{ { x } , { x , y } }` (Kuratowski pair), which has one or two members, but there is no such thing as a "first" or "second" one.